### PR TITLE
add new features to preview SDK

### PR DESCRIPTION
**Pull request by jadrefke**

### CODE REVIEW

File: sdk/search/Microsoft.Azure.Search.Service/src/Generated/SdkInfo_SearchServiceClient.cs [hunk 1]
```diff
@@ -28,16 +28,5 @@ internal static partial class SdkInfo
...
-      // BEGIN: Code Generation Metadata Section
-      public static readonly String AutoRestVersion = "latest";
-      public static readonly String AutoRestBootStrapperVersion = "autorest@2.0.4407";
-      public static readonly String AutoRestCmdExecuted = "cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/search/data-plane/Microsoft.Azure.Search.Service/readme.md --csharp --version=latest --reflect-api-versions --tag=track1-package-2019-05-preview --csharp-sdks-folder=D:\\src\\azure-sdk-for-net\\sdk";
-      public static readonly String GithubForkName = "Azure";
-      public static readonly String GithubBranchName = "master";
-      public static readonly String GithubCommidId = "2222f0ff6009aaf9aa632af5a678f2f874dff6fd";
-      public static readonly String CodeGenerationErrors = "";
-      public static readonly String GithubRepoName = "azure-rest-api-specs";
-      // END: Code Generation Metadata Section
```
Comment (bleroy) [inline sdk/search/Microsoft.Azure.Search.Service/src/Generated/SdkInfo_SearchServiceClient.cs]: Why is this disappearing?

**Review by bleroy — APPROVED** [review]
Pending final approval and merge of associated spec PR, and incorporation into this PR of any changes associated with the CR over there. https://github.com/Azure/azure-rest-api-specs/pull/7812

**Comment by brjohnstmsft** [comment]
This change requires test coverage.

**Review by bitsforthought — CHANGES_REQUESTED** [review]
Test coverage is missing.

**Pull request closed without merging**